source/ceylon/time/internal/AbstractDate.ceylon

```diff
@@ -7,22 +7,17 @@ shared abstract class AbstractDate( dayOfEra )
     "Every [[Date]] implementation should indicate it´s own _day of era_ based in it´s own chronology."
     shared actual Integer dayOfEra;
 
-    //TODO: How to deal with two dates of different chronology, maybe compare should use _Rata Die_?
     "Dates from same chronology can be compared if they have same _day of era_."
     shared actual Comparison compare(Date other) {
         return dayOfEra <=> other.dayOfEra;
     }
 
-    //TODO: How to deal with two dates of different chronology, maybe compare should use _Rata Die_?
-    "Dates from same chronology can be considered equal if they have same _day of era_."
+    "Two `Date`s are considered equals if they have the same 
+     _day of era_."
     shared actual Boolean equals( Object other ) {
-        if (is AbstractDate other) {
-            return dayOfEra.equals(other.dayOfEra);
-        }
-        return false;
+        return (super of Date).equals(other);
```

Comment:
right, Ill change the implementations to extend from `Object` instead of `Basic`